### DISCUSSION TITLE
Welcome to CyberSecurity Discussions!

### DISCUSSION THREAD

**Discussion by Berkanktk** [Berkanktk/CyberSecurity#4] [Announcements]
## 👋 Hello fellow hackers!
I'm using Discussions as a place to connect with other members of the cybersecurity community, so feel free to create a new discussion whenever you want. 

I'll also be appreciating any discussions regarding:
* Potential topics that should be included in the repo.
* Feedback on how i could improve the structure -> improving the overall experience when browsing.
* Errors, misspellings and other issues you encounter.
* Want me or other people in the community to explain something unclear in the repo.

Remember that this is a community we build together, so let's learn and improve ourselves through passing our knowledge to other people. 💪

**Comment by Lynurban** [comment]
I'm interested in learning about cybersecurity. Could you guide me on the best direction to go about it? I'm still a novice here.

**Reply by Berkanktk** [reply]
Hi there! I would definitely take a look at [TryHackMe ](https://tryhackme.com/signup?referrer=603d97391f24592a9418f6e0) and [HackTheBox Academy](https://academy.hackthebox.com/) to begin with. Especially the first one, since it has some incredible beginner friendly rooms you can play around with. :)

**Reply by Lynurban** [reply]
@Berkanktk Thank you
